plugins/kubernetes/src/types/types.ts

```diff
@@ -46,4 +46,4 @@ export interface ClusterLinksFormatterOptions {
 
 export type ClusterLinksFormatter = (
   options: ClusterLinksFormatterOptions,
-) => URL;
+) => string;
```

Comment:
Why did you change this contract for all formatters? Seems like a separate discussion to take.